.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: required
+language: node_js
+services:
+- docker
+os:
+- linux
+node_js:
+- '8'
+git:
+  depth: false
+cache:
+  directories:
+  - ".git"
+before_install:
+- openssl aes-256-cbc -k ${GCR_KEY} -in key.json.enc -out key.json -d
+- npm i @npm-wharf/shipwright@latest -g
+- npx makeshift -s @npm -r https://registry.internal.npmjs.com
```

Comment:
Do we need to npx this? It requires a whole separate install sequence to echo one line

Reply:
this travis yml is from the other repos -- just copied wholesale.  at least it's consistent?